src/main/scala/com/github/unchama/seichiassist/subsystems/shareinventory/domain/ShareInventoryPersistence.scala

```diff
@@ -0,0 +1,28 @@
+package com.github.unchama.seichiassist.subsystems.shareinventory.domain
+
+import com.github.unchama.seichiassist.subsystems.shareinventory.domain.bukkit.InventoryContents
+
+import java.util.UUID
+
+trait ShareInventoryPersistence[F[_]] {
+
+  /**
+   * [[InventoryContents]]をセーブします。
+   * @param inventoryContents セーブ対象の[[InventoryContents]]
+   */
+  def saveSerializedShareInventory(
```

Comment:
```suggestion
  def save(
```

`Serialized`...メモリ上にデータが存在しているので矛盾しています
`ShareInventory`...nit: 自明なので省略してもよいのでは？ (以下、このファイル内では同様)